app/models/user.rb

```diff
@@ -17,6 +17,18 @@ class User < ActiveRecord::Base
     message: I18n.t("validations.accepted"),
   }
 
+  def self.admin
+    where(admin: true)
+  end
+
+  def self.cyclist
+    where("users.assigned_zone_id IS NOT NULL")
```

Comment:
What are your thoughts on using AREL selector like `where.not(assigned_zone_id: nil)`